jscomp/main/jsoo_main.ml

```diff
@@ -97,7 +97,7 @@ let implementation ~use_super_errors prefix impl str  : Js.Unsafe.obj =
     let ast = impl 
       (Lexing.from_string
         (if prefix then "[@@@bs.config{no_export}]\n#1 \"repl.ml\"\n"  ^ str else str )) in 
-    let ast = Reactjs_jsx_ppx_v3.rewrite_implementation ast in 
+    let ast = Reactjs_jsx_ppx_v3.rewrite_implementation ?version:react_ppx_version ast in 
     let ast = Bs_builtin_ppx.rewrite_implementation ast in 
```

Comment:
Ah, I meant we don't need touch reactjs_jsx_ppx.cppo. Instead we do something like this:
```
let ast = match version with 
   | None -> ast 
   | V2 -> Reactjs_jsx_ppx_v2.rewrite_implementation xxx
  | V3 -> Reactjs_jsx_ppx_v3.rewrite_implementation xxx 
```

Reply:
Won't that make the bundle larger than necessary? We would have two versions of the same ppx in it.

Reply:
Talked offline - decided the bundle size was small enough that it was worth the tradeoff.